### DISCUSSION TITLE
Resilio Sync

### DISCUSSION THREAD

**Discussion by sebortner** [selfhst/icons#121] [New Icon Request]
![Resilio](https://github.com/user-attachments/assets/ad505d45-203e-403b-be50-e96a6adc7f07)

Edit: I just realized, they might have changed their Logo for "Sync". https://www.resilio.com/sync/

**Comment by shollyethan** [comment]
Resilio Sync has been added using the latest logo - thanks!

**Reply by sebortner** [reply]
Whoa, that was quick. Thanks!
